penguin/app/components/looksgood.js

```diff
@@ -62,6 +62,18 @@ class LooksGood extends Component{
   handleLookingGood(){
     // do fetch POST request that includes photo
     // photo is at filepath described in this.props.picture
+    // 
+    fetch('http://localhost:8000/photos', {
+      method: 'POST',
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        uri: this.props.picture,
```

Comment:
@kevinwin Are we assuming that the user is going to provide a photo uri? Or is that just how the camera feature works on iphone/React Native?